src/DynamoUtilities/TestUtilities.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DynamoUtilities
+{
+    // Bag of utilities designed for use during tests
+    internal static class TestUtilities
+    {
+        // Simple string that we can store in DynamoWebView2 instances so that we can track them down more easily
+        internal static string WebView2Stamp;
```

Comment:
I'm having a hard time understanding why it's useful to make this static... why not just give the DynamoWebView class an id at startup?

Reply:
It is useful to have it as a static so that we can access more easily from all over Dynamo (without being linked to the DynamoCoreWpf project)

Reply:
couldn't you solve this with an interface in DynamoServices? (also sorry for not seeing your reply).

Reply:
Maybe, but it did not seem simpler than a static meant for use only during testing. I might switch to something directly done in the webview2 class constructor